nesta/production/orms/nomis_orm.py

```diff
@@ -0,0 +1,110 @@
+'''
+ONS NOMIS ORM
+=======================
+
+The model for ONS's NOMIS data.
+
+'''
+
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.mysql import VARCHAR, TEXT
+from sqlalchemy.types import INT, DATETIME, FLOAT
+from sqlalchemy import Column
+
+Base = declarative_base()
+
+class NomisGeographyLookup(Base):
+    __tablename__ = 'nomis_geography_lookup'
+
+    code = Column(VARCHAR(9), primary_key=True)
+    name = Column(TEXT)
+
+
+class NomisIndustryLookup(Base):
+    __tablename__ = 'nomis_industry_lookup'
+
+    code = Column(VARCHAR(3), primary_key=True)
+    name = Column(TEXT)
+
+
+class NomisEmploymentSizebandLookup(Base):
+    __tablename__ = 'nomis_employment_sizeband_lookup'
+
+    code = Column(INT, primary_key=True, autoincrement=False)
+    name = Column(TEXT)
+
+
+class NomisBusinessCount(Base):
+    __tablename__ = 'nomis_businesscount'
+
+    date = Column(DATETIME, primary_key=True, index=True)
+    employment_sizeband_code = Column(INT, primary_key=True, index=True, autoincrement=False)
+    geography_code = Column(VARCHAR(9), primary_key=True, index=True)
+    industry_code = Column(VARCHAR(3), primary_key=True, index=True)
+    obs_value = Column(INT)
+
+
+class NomisVariableLookup(Base):
+    __tablename__ = 'nomis_variable_lookup'
+
+    code = Column(INT, primary_key=True, autoincrement=False)
+    name = Column(TEXT)
+
+
+class NomisEmployment(Base):
+    __tablename__ = 'nomis_employment'
+
+    date = Column(DATETIME, primary_key=True, index=True)
+    geography_code = Column(VARCHAR(9), primary_key=True, index=True)
+    obs_value = Column(INT)
+    variable_code = Column(INT, primary_key=True, index=True, autoincrement=False)
+
+
+# class NomisPopulationCensus(Base):
+#     __tablename__ = 'nomis_population_census'
+
+#     date = Column(INT, primary_key=True, index=True)
+#     indgepuk11_code = Column(VARCHAR(1), primary_key=True, index=True)
+#     geography_code = Column(VARCHAR(9), primary_key=True, index=True)
+#     indgepuk11_name = Column(VARCHAR(130))
+#     obs_value = Column(INT)
```

Comment:
This can be deleted or is being saved for potential future use?

Reply:
It turns out the Scotland runs it's own census, so the NOMIS census isn't needed yet. We *could* end up using in future for other work, so I thought no point removing it all together.